homeassistant/components/zodiac/__init__.py

```diff
@@ -21,3 +22,14 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
     )
```

Comment:
Please start the import step in the config flow from here and remove the call to `async_load_platform`.